openmc/mesh.py

```diff
@@ -1137,6 +1137,70 @@ def from_hdf5(cls, group):
 
         return mesh
 
+    @classmethod
+    def from_domain(
+        cls,
+        domain,
+        dimension=[100, 100, 100],
```

Comment:
Mutable default arguments are generally [a bad idea](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)
```suggestion
        dimension=(100, 100, 100),
```
Also, this seems like a pretty fine mesh (1 million total elements). Maybe the default should be a bit coarser?

Reply:
Alternatively, make `dimension` a required argument rather than having an arbitrary default

Reply:
I can make the default courser, the same default of [100, 100, 100] 
also as a mutable list was made [here]( [100, 100, 100] ) so i can change that as well